tools/distrib/python/grpcio_tools/grpc_tools/command.py

```diff
@@ -55,11 +58,11 @@ def build_package_protos(package_root, strict_mode=False):
     for proto_file in proto_files:
         command = [
             "grpc_tools.protoc",
-            "--proto_path={}".format(inclusion_root),
+            "--proto_path={}".format(relative_path),
```

Comment:
we should probably keep `inclusion_root` variable name here, just set it to what we now have in `relative_path`